test/unit/redirector_test.rb

```diff
@@ -1,91 +0,0 @@
-require 'test_helper'
```

Comment:
should we keep the tests to make sure we did the parameterization of the gem correctly? 
I dont think we should test the gems functionality, however I am also not sure how it works, and our code was working pretty fine before and it was well tested.